docs/MO_DG/prepare_model/Additional_Optimizations.md

```diff
@@ -2,149 +2,237 @@
 
 @sphinxdirective
 
-Input data for inference can be different from the training dataset and requires 
-additional preprocessing before inference. To accelerate the whole pipeline including 
-preprocessing and inference, Model Optimizer provides special parameters such as ``--mean_values``,
-
-``--scale_values``, ``--reverse_input_channels``, and ``--layout``. Based on these 
-parameters, Model Optimizer generates OpenVINO IR with additionally inserted sub-graphs 
-to perform the defined preprocessing. This preprocessing block can perform mean-scale 
-normalization of input data, reverting data along channel dimension, and changing 
-the data layout. See the following sections for details on the parameters, or the 
-:doc:`Overview of Preprocessing API <openvino_docs_OV_UG_Preprocessing_Overview>` 
+Input data for inference can be different from the training dataset and requires
+additional preprocessing before inference. To accelerate the whole pipeline including
+preprocessing and inference, Model Conversion API provides special parameters such as ``mean_values``,
```

Comment:
Do we have any rules when to use `mo Python API` or `Model Conversion API`? How do we select it?

Reply:
MO Python API and Model Conversion API refers to convert_model() method, so it is generally the same thing. But we do not want to use the term Model Optimizer (MO), as convert_model() does not optimize the model and it is a legacy naming. So preferable is to use Model Conversion API. If we talk about Model Optimizer command line tool, which has historically Model Optimizer name, we can use the term "MO command line tool", but not "Model Optimizer command line tool".